modules/wallet/ui/ConnectWalletModal/ConnectWalletModal.tsx

```diff
@@ -1,41 +1,37 @@
-import { useCallback } from 'react'
-import { Modal, ModalProps } from '@lidofinance/lido-ui'
-import { ConnectWalletModalTerms } from './ConnectWalletModalTerms'
-import {
-  ConnectMetamaskButton,
-  ConnectWalletConnectButton,
-  ConnectLedgerButton,
-  // ConnectCoinbaseButton,
-  // ConnectTrustButton,
-  // ConnectImTokenButton,
-} from '../ConnectButton'
-import { useLocalStorage } from 'modules/shared/hooks/useLocalStorage'
-import { STORAGE_TERMS_KEY } from 'modules/config'
+import { useThemeToggle } from '@lidofinance/lido-ui'
 
-type Props = ModalProps & {}
+import { WalletsModalForEth } from '@reef-knot/connect-wallet-modal'
 
-export function ConnectWalletModal(props: Props) {
-  const { onClose } = props
-  const [checked, setChecked] = useLocalStorage(STORAGE_TERMS_KEY, false)
+type WalletModalForEthProps = React.ComponentProps<typeof WalletsModalForEth>
+
+const HIDDEN_WALLETS: WalletModalForEthProps['hiddenWallets'] = [
+  'Coinbase',
+  'Trust',
+  'ImToken',
+  'Coin98',
+  'MathWallet',
+  'Tally',
+  'Ambire',
+  'Blockchain.com Wallet',
+  'ZenGo',
+  'Brave Wallet',
+  'Opera Wallet',
+  'Exodus',
+  'Gamestop',
+  'Xdefi',
+  'Zerion',
+]
 
-  const handleChange = useCallback(() => {
-    setChecked(currentValue => !currentValue)
-  }, [setChecked])
+type Props = WalletModalForEthProps & {}
 
-  const common = {
-    disabled: !checked,
-    onConnect: onClose,
-  }
+export function ConnectWalletModal(props: Props) {
+  const { themeName } = useThemeToggle()
```

Comment:
In easy tracks, there is no theme switching =/

Reply:
But if it will be in future, we will not miss this feature here